doc/sphinx/addendum/extraction.rst

```diff
@@ -99,12 +99,15 @@ Extraction Options
 Setting the target language
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. cmd:: Extraction Language {| OCaml | Haskell | Scheme }
+.. cmd:: Extraction Language {| OCaml | Haskell | Scheme | JSON }
```

Comment:
Can you mark issue #10902 to be closed? Or does @Zimmi48 have a plan for this issue?

Reply:
Oh, I didn’t realise that it was already reported. When I saw that there was a `JSON` output, I instinctively added it in the documentation, as it did help me a lot to understand how to update the extraction! ☺

Reply:
Nope. Thanks for fixing this issue!